pygletExample.py

```diff
@@ -0,0 +1,469 @@
+import math
+import sys
+import threading
+
+import IPython
```

Comment:
Ipython will need proper referencing due to nature of license

Reply:
referencing through the pip install should be sufficient as we are only calling these libraries